timelock-impl/src/main/java/com/palantir/atlasdb/timelock/management/DisabledNamespaces.java

```diff
@@ -57,17 +60,19 @@ public static DisabledNamespaces create(DataSource dataSource) {
 
     private void initialize() {
         execute(Queries::createTable);
+        disabledNamespaces = disabledNamespaces();
```

Comment:
The javadoc of `Collectors.toSet()` says we don't get guarantees on mutability nor thread-safety, so we can't really do the adds or removes later.

Also, do we expect disable and enable to need to be threadsafe?

Reply:
fix for modifiiability:
`                .collect(Collectors.toCollection(HashSet::new));`

Reply:
yeah just commented this offline

Reply:
`Collectors.toCollection(Sets::newConcurrentHashSet)`